### PR TITLE
chore!: update dashmate config for v0.25.20

### PR DESCRIPTION
**Pull request by shumkov**
## Issue being fixed or feature implemented

We need to update dashmate config to represent changes in v0.25.20

## What was done?

- Added new configuration options and corresponding variables
 

## How Has This Been Tested?

None

## Breaking Changes

Dashmate < 0.25.20 is not supported

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: ansible/roles/dashmate/templates/dashmate.json.j2 [hunk 1]
```diff
@@ -213,7 +224,25 @@
             },
             "consensus": {
               "createEmptyBlocks": true,
-              "createEmptyBlocksInterval": "3m"
+              "createEmptyBlocksInterval": "3m",
+              "peer": {
+                "gossipSleepDuration": "{{dashmate_platform_drive_tenderdash_consensus_peer_gossip_sleep_duration}}",
+                "queryMaj23SleepDuration": "{{dashmate_platform_drive_tenderdash_consensus_peer_query_maj23_sleep_duration}}"
+              },
+              "unsafeOverride": {
```
Comment (lklimek) [inline ansible/roles/dashmate/templates/dashmate.json.j2:232]: Why not using consensus params instead of overrides?

Comment (shumkov) [reply ansible/roles/dashmate/templates/dashmate.json.j2:232]: We can't change the consensus param without resetting the network. Having these options we can experiment with numbers quickly. It's not for persistent configuration.

**Review by QuantumExplorer — APPROVED** [review]

File: ansible/roles/dashmate/templates/dashmate.json.j2 (hunk 1 above)
Comment (QuantumExplorer) [reply ansible/roles/dashmate/templates/dashmate.json.j2:232]: Acually we can chance the consensus params once an hour on testnet.

Comment (QuantumExplorer) [reply ansible/roles/dashmate/templates/dashmate.json.j2:232]: But I agree this is faster for now.

Comment (shumkov) [reply ansible/roles/dashmate/templates/dashmate.json.j2:232]: You mean if we hardcode this change versions?

**Pull request merged**